rss2email/email.py

```diff
@@ -192,7 +192,7 @@ def imap_send(message, config=None, section='DEFAULT'):
             username = config.get(section, 'imap-username')
             password = config.get(section, 'imap-password')
             try:
-                if not ssl:
+                if ssl:
```

Comment:
The original code is [right here][1].  The current logic doesn't provide a way to send login credentials in the clear, or to enable STARTTLS without sending credentials.  If you need either of those, file a new bug explaining why.

[1]: https://github.com/wking/rss2email/blob/master/README#L188-L192